discovery-provider/alembic/env.py

```diff
@@ -121,10 +124,7 @@ def run_migrations_online():
         with context.begin_transaction():
             context.run_migrations()
 
-        try:
-            load_triggers(connection)
-        except Exception as e:
-            print(e)
```

Comment:
Removed the try / catch here so errors bubble up... but that causes `test_migration_idempotency` to fail.

Should I: re-add the try / catch or disable `test_migration_idempotency`?

Reply:
I think for now re-add the try catch? I think that test is quite useful to have

And it seems to fail for unrelated
```
E       psycopg2.errors.UndefinedTable: relation "users" does not exist
E       CONTEXT:  SQL statement "alter table users
E             add column if not exists is_available boolean not null default true"
```

Shouldn't the table exist?

Reply:
The `test_migration_idempotency` starts at the beginning and runs each migration twice... but the `env.py` will attempt to run ddl.sql after each alembic run... for an empty database there is no users table and it fails.

I think this indicates the try / catch is actually necessary for running older migrations...